### PR TITLE
[4.0] Module Assignment [a11y]

### PR DESCRIPTION
**Pull request by brianteeman**
When you assign a module to a menu item there is a select for On All Pages, Selected etc

image

If you are on the either of the two "selected" options you will see a search box and a tree.

1. The search input does not have a label - for a11y it must have one - this PR adds aria-label attribute to resolve that

2. The search input has tabindex-1 which means that it is not part of the navigation tree and you can never _tab_ into the input to perform a search - this PR removes the tabindex so you can now tab into the search input

### CODE REVIEW

File: administrator/components/com_modules/tmpl/module/edit_assignment.php [hunk 1]
```diff
@@ -48,7 +48,8 @@
 					<a id="treeCollapseAll" href="javascript://"><?php echo Text::_('JNONE'); ?></a>
 				</span>
 				<input type="text" id="treeselectfilter" name="treeselectfilter" class="form-control search-query float-right" size="16"
-					autocomplete="off" placeholder="<?php echo Text::_('JSEARCH_FILTER'); ?>" aria-invalid="false" tabindex="-1">
+					autocomplete="off" placeholder="<?php echo Text::_('JSEARCH_FILTER'); ?>"
+					aria-invalid="false" aria-label=<?php echo Text::_('JSEARCH_FILTER'); ?>>
```
Comment (Quy) [inline administrator/components/com_modules/tmpl/module/edit_assignment.php]:
```suggestion
					aria-invalid="false" aria-label="<?php echo Text::_('JSEARCH_FILTER'); ?>">
```

File: administrator/components/com_modules/tmpl/module/edit_assignment.php [hunk 2]
```diff
@@ -48,7 +48,8 @@
 					<a id="treeCollapseAll" href="javascript://"><?php echo Text::_('JNONE'); ?></a>
 				</span>
 				<input type="text" id="treeselectfilter" name="treeselectfilter" class="form-control search-query float-right" size="16"
```
Comment (Quy) [inline administrator/components/com_modules/tmpl/module/edit_assignment.php:50]:
```suggestion
				<input type="text" id="treeselectfilter" name="treeselectfilter" class="form-control search-query"
```

Comment (brianteeman) [reply administrator/components/com_modules/tmpl/module/edit_assignment.php:50]: please create your own pr for this

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3506d838bb7435707127080455671657911c6cf3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27445">issues.joomla.org/tracker/joomla-cms/27445</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 3506d838bb7435707127080455671657911c6cf3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27445">issues.joomla.org/tracker/joomla-cms/27445</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27445">issues.joomla.org/tracker/joomla-cms/27445</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
w00t

**Pull request merged**